index.js

```diff
@@ -40,9 +40,9 @@ client.on('ready', () => {
 
 client.on('message', (message) => {
 	if (
-		!message.content.startsWith(prefix) ||
+		(!message.content.startsWith(prefix)) ||
 		message.author.bot ||
-		message.channel.type == 'dm'
+		message.channel.type == 'DM'
```

Comment:
Could you please explain the reason behind this update?

Reply:
For some reason it was replying in dms... I checked the docs and they have uppercase specifications for this... so I thought to fix the bug...

Reply:
But we want the replies in DM :thinking:

Reply:
check my comment on the original issue

Reply:
Ah, got it. So will this fix the issue in #73 ?

Reply:
Ah Got it the error is not this.. I found the error I'll make another commit to this pr... that should fix it

Reply:
Now it doesn't reply in the dms.. Tested. For some reason not enclosing it into brackets was the bug